src/starcounter-debug-aid.html

```diff
@@ -173,6 +82,12 @@
                 //this.$.errorDialog.reset();
             },
             openDialog: function () {
+                if (!Polymer.isInstance(this.$.ui)) {
+                    var fileName = "starcounter-debug-aid.html";
+                    var link = document.querySelector('link[href$="' + fileName + '"]');
+                    var baseUrl = link.href.replace(fileName, '');
+                    Polymer.Base.importHref(baseUrl + "starcounter-debug-aid-ui.html");
```

Comment:
I created an issue for this https://github.com/Starcounter/starcounter-debug-aid/issues/44